### DISCUSSION TITLE
Constraint Flipping Value Error

### DISCUSSION THREAD

**Discussion by nbungart1999** [phoebe-project/phoebe2#489] [Questions]
Does anyone know what is causing the error here? I have looked over this multiple times but I can't seem to find anything wrong or how to fix it. The error message points at the adding the constraint line but in the error message it talks about one of the twigs: 'requiv@' but that's about all I have.
![image](https://user-images.githubusercontent.com/83960306/125102904-c8ecf780-e0a9-11eb-8ffb-0ad3bc2ab7b6.png)
![image](https://user-images.githubusercontent.com/83960306/125102933-d1ddc900-e0a9-11eb-801d-b1cc30eb88d7.png)

**Comment by kecnry** [comment]
this should have been fixed in the 2.3.45 bugfix release (#475), but it seems it didn't address all cases.  If you're not using 2.3.45 or later, please update phoebe as that may fix your case here (I can't tell without the rest of your script).  I'll work on getting a bugfix out soon for the remaining cases, so keep an eye out for that release.

**Reply by nbungart1999** [reply]
What's the command to see what version of phoebe I have?

**Reply by kecnry** [reply]
`print(phoebe.__version__)`

**Reply by nbungart1999** [reply]
Ok thanks! It looks like I was on 2.3.44 so I just upgraded my version. I'll let you know if the issue is resolved when I run my script again!

**Reply by nbungart1999** [reply]
Works fine now! The fix works fine and the bug no longer happens

**Reply by kecnry** [reply]
Great!  I'll still work on getting the full bugfix out (a similar bug can happen if you flip things in different combinations), but its good to know no one is currently waiting on it.

**Reply by kecnry** [reply]
The remaining cases should now be fixed in 2.3.49 (#490)
